helios-service-registration/src/main/java/com/spotify/helios/serviceregistration/ServiceRegistration.java

```diff
@@ -24,6 +24,7 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import com.google.common.base.Objects;
```

Comment:
Good catch. I moved the imports around manually, and forgot to clean them up. Will push another commit.